x/validator/keeper/msg_server_approve_disable_validator.go

```diff
@@ -54,13 +63,19 @@ func (k msgServer) ApproveDisableValidator(goCtx context.Context, msg *types.Msg
 
 	// check if proposed disable validator has enough approvals
 	if len(proposedDisableValidator.Approvals) == k.DisableValidatorApprovalsCount(ctx) {
+		_, isFound := k.GetRejectedNode(ctx, proposedDisableValidator.Address)
+		if isFound {
+			k.RemoveRejectedNode(ctx, proposedDisableValidator.Address)
```

Comment:
We've already removed the rejected node in the ProposeDisable msg handler, haven't we?